modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/PartitionReplicaLifecycleManager.java

```diff
@@ -1465,6 +1465,8 @@ private CompletableFuture<Void> changePeersOnRebalance(
                                                 .exceptionally(e -> null);
```

Comment:
We skip exceptions here and also do not log anything in outer `thenCompose` methods, I find this suspicious.

Reply:
any exceptions that can occur inside these futures

Reply:
Fixed.